test/qa/test_connect6.py

```diff
@@ -18,14 +18,12 @@ def setup_function(function):  # noqa: ARG001
     daemon.start()
     login.login_as("default")
     logging.log()
-    print(sh.nordvpn.set.ipv6.on())
```

Comment:
this removal makes the `connect6` tests into duplicated connect tests, and we are not running these in CI either. imo either remove the whole files or leave them be as is to be decided what we do with them later